packages/rehype-mathjax/index.d.ts

```diff
@@ -0,0 +1,25 @@
+import {Plugin} from 'unified'
+
+// Should be ported back to MathJax repo
+// http://docs.mathjax.org/en/latest/options/output/svg.html#the-configuration-block
+interface MathJaxSVGOptions {
+  scale: number
+  minScale: number
+  mtextInheritFont: boolean
+  merrorInheritFont: boolean
+  mathmlSpacing: boolean
+  skipAttributes: {[index: string]: boolean}
+  exFactor: number
+  displayAlign: 'left' | 'center' | 'right'
+  displayIndent: string
+  fontCache: 'local' | 'global'
+  localID: string | null
+  internalSpeechTitles: true
+  titleID: number
+}
```

Comment:
I'm cautious about maintaining a copy of the MathJax typings.
I'd lean towards getting this added upstream, them incorporating the update here.

thoughts @wooorm ? :thought_balloon:

Reply:
It would be great if we can import them, indeed!

Reply:
@ChristianMurphy @wooorm Yes it should definitely be bundled in mathjax-full package. I'll open PR on MathJax repo to include these interfaces on their release.

Reply:
@ChristianMurphy I initially thought I would restart working on this PR after [this issue](https://github.com/mathjax/MathJax-src/issues/524) resolved, but there's no response for a long time. Is it ok to temporary bundle those interfaces into remark-math until it is included in MathJax?

Reply:
Yeah, I think so!